nixos/modules/module-list.nix

```diff
@@ -1312,6 +1312,7 @@
   ./services/video/epgstation/default.nix
   ./services/video/go2rtc/default.nix
   ./services/video/frigate.nix
+  ./services/video/iSponsorBlockTV.nix
```

Comment:
Thanks for the contribution, but this file doesn't exist, why was this line added?